### PR TITLE
[WIP] ngeo-desktop-geolocation

### PR DESCRIPTION
**Pull request by adube**
This PR introduces the `ngeo-desktop-geolocation` directive.

You can wait for #420 to be fully reviewed before reviewing this one as both have lots of things in common.

### CODE REVIEW

File: externs/ngeox.js [hunk 1]
```diff
@@ -327,6 +327,39 @@ ngeox.MeasureEvent.prototype.feature;
...
 
 /**
+ * Options for the mobile geolocations directive.
+ * @typedef {{
+ *    accuracyFeatureStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    positionFeatureStyle: (ol.style.Style|Array.<ol.style.Style>|ol.style.StyleFunction|undefined),
+ *    zoom: (number|undefined)
+ * }}
+ */
+ngeox.DesktopGeolocationDirectiveOptions;
```
Comment (fgravin) [inline externs/ngeox.js]: Could'nt you have a single type for mobile and desktop or not ?
Do they differ, or could differ in the future ?

**Comment by fgravin** [comment]
Please update your code with the comments i've made in the mobile geolocation PR.

File: externs/ngeox.js (hunk 1 above)
Comment (adube) [inline externs/ngeox.js]: Excellent point.  I went that single type approach at first, then chose to have them separated even if they have the same options.  They are currently exactly the same.  I don't have a strong opinion on this matter.  Let me know which one you prefer.

**Comment by adube** [comment]
I'm on it.

**Comment by adube** [comment]
gh-page: http://adube.github.io/ngeo/gmf-geolocation-desktop/examples/desktopgeolocation.html

**Comment by adube** [comment]
I think this is ready for merge too, once travis completes its check.

**Comment by adube** [comment]
This is ready for merge too.

File: externs/ngeox.js (hunk 1 above)
Comment (fgravin) [inline externs/ngeox.js]: Ok to leave this this way for now.

**Comment by fgravin** [comment]
I will merge it but i have a little concern about this approach.
We will have no reference on the geolocation object outside the directive.

**Pull request merged**